ipatests/test_integration/test_installation.py

```diff
@@ -389,6 +389,16 @@ def test_install_dns(self):
         tasks.install_dns(self.master)
 
 
+    def test_WSGI_worker_process(self):
+        """ Test if WSGI worker process count is set to 4
+
+        related ticket : https://pagure.io/freeipa/issue/7587
+        """
+        master = self.master
```

Comment:
You do not need this variable, you are using self.master only once.

And BTW this file will not exists if FreeIPA is not installed:
http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/ecab09d6-9efe-11e8-a848-fa163e15adc6/runner.log.gz